tests/exporter/collectors/test_network.py

```diff
@@ -0,0 +1,32 @@
+import unittest
+from exporter.collectors.network import NetworkCollector
+
+class TestNetworkCollector(unittest.TestCase):
```

Comment:
### Requested Changes
* Add the following two tests for the Traffic In and Traffic Out:
  * `test_get_traffic_in_is_not_negative` :  check if method returns >= 0
  * `test_get_traffic_out_is_not_negative` : check if method returns >= 0